src/directives/visibility.directive.ts

```diff
@@ -14,50 +15,26 @@ import {
  *
  */
 @Directive({ selector: '[visibilityObserver]' })
-export class VisibilityDirective implements OnInit, OnDestroy {
+export class VisibilityDirective {
```

Comment:
Whats the point of this directive now you removed loop check?

Reply:
Not really sure it's needed anymore. It can be collapsed back into the main datatable.

Reply:
Does it work w/ the hidden examples w/o it?